DigitalLearningSolutions.Web.Tests/Helpers/DisplayStringHelperTests.cs

```diff
@@ -310,5 +310,21 @@ public void GetTimeStringFromMinutes_returns_expected_result(int minutes, string
             // Then
             result.Should().Be(expectedResult);
         }
+
+        [Test]
+        [TestCase("", "")]
+        [TestCase("NicePascalCaseString", "Nice Pascal Case String")]
+        [TestCase("ASlightlyMoreAwkwardOne", "A Slightly More Awkward One")]
+        [TestCase("XMLFile", "XML File")]
+        [TestCase("An Already Separated String", "An Already Separated String")]
+        [TestCase("A Partially SeparatedString", "A Partially Separated String")]

```

Comment:
Does it work for strings starting with 'ID' or which have 'XML' not at the start? (if not, how exhaustive do we want to get? are we even worried about those cases?)